src/CLUBB_core/clubb_api_module.F90

```diff
@@ -4534,7 +4546,10 @@ subroutine set_default_clubb_config_flags_api( iiPDF_type, & ! Out
                                                  l_mono_flux_lim_rtm, & ! Out
                                                  l_mono_flux_lim_um, & ! Out
                                                  l_mono_flux_lim_vm, & ! Out
-                                                 l_mono_flux_lim_spikefix ) ! Out
+                                                 l_mono_flux_lim_spikefix, & !Out 
+                                                 l_modify_ic_for_cnvg_test, & !Out
```

Comment:
It looks like the pull requests are mixing together code.  The `l_modify_ic_for_cnvg_test` from https://github.com/larson-group/clubb_release/pull/4 appears in this pull request.  Is there a way to keep the code changes from the pull requests separate?  Should we merge in the pull requests in a particular order?  If so, what is that order?

Reply:
@vlarson : It is my bad, let me modify and redo the pull request